server/app/auth/oidc/admin/AdfsProfileAdapter.java

```diff
@@ -65,17 +65,17 @@ protected void adaptForRole(CiviFormProfile profile, ImmutableSet<Roles> roles)
     }
   }
 
+  @SuppressWarnings("unchecked")
+  // profile.getAttribute returns List<Object>. There's no way to make it type safe.
   private boolean isGlobalAdmin(OidcProfile profile) {
-    JSONArray groups = (JSONArray) null;
-    if (profile.containsAttribute(this.ad_groups_attribute_name)) {
-      groups = profile.getAttribute(this.ad_groups_attribute_name, JSONArray.class);
-    }
-
+    ImmutableList<Object> groups =
+        profile.containsAttribute(this.ad_groups_attribute_name)
+            ? profile.getAttribute(this.ad_groups_attribute_name, ImmutableList.class)
```

Comment:
getAttribute returns an Object not a List so I'd expect this to fail

BasicUserProfile
public Object getAttribute(final String name)